src/lib/containers/home_page/featured-data/FacetPlotsCard.tsx

```diff
@@ -116,7 +116,7 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
       </div>
     }
     const isShowingMultiplePlots = facetPlotDataArray.length > 1
-    const cardTitle = title ?? isShowingMultiplePlots ? selectedFacetValue : unCamelCase(facetDataArray[0].columnName, facetAliases)
+    const cardTitle = title ?? (isShowingMultiplePlots ? selectedFacetValue : unCamelCase(facetDataArray[0].columnName, facetAliases))
```

Comment:
Also noticed that the card title was wrong when a title was provided, because of the order of operations.  Made explicit (that ternary operator statement is the fallback value, if the title is undefined/null)